### PR TITLE
Feature/155 display venue name

### PR DESCRIPTION
**Pull request by jmarx**
#155 

Display the venue name in the event list block.

### CODE REVIEW

File: core/classes/class-rest-api.php [hunk 1]
```diff
@@ -316,6 +316,7 @@ function( $slug ) {
 		if ( $query->have_posts() ) {
 			foreach ( $query->posts as $post_id ) {
 				$event   = new Event( $post_id );
+				$venue_information = $event->get_venue_information();
```
Comment (jmarx) [inline core/classes/class-rest-api.php:319]: @jmarx Maybe set a default in case this function fails for whatever reason?

**Comment by pbrocks** [comment]
@jmarx Can you share what you are seeing with some screenshots?

**Comment by jmarx** [comment]
@pbrocks I have to wait until later(when I turn off my work VPN) But I can tell you that I just get the post title, but don't see any events when I save a post with the events list block. Like I said in slack, It's probably some silly issue like permalinks or something. I'll follow up later.

File: core/classes/class-rest-api.php (hunk 1 above)
Comment (jmarx) [reply core/classes/class-rest-api.php:319]: As a follow-up, I realize that this is not needed as that `get_venue_information` method returns and empty venue by default.

File: core/classes/class-rest-api.php [hunk 2]
```diff
@@ -327,6 +328,7 @@ function( $slug ) {
 					'featured_image_thumbnail' => get_the_post_thumbnail( $post_id, 'thumbnail' ),
 					'attendees'                => ( $event->attendee ) ? $event->attendee->attendees() : array(),
 					'current_user'             => ( $event->attendee && $event->attendee->get( get_current_user_id() ) ) ? $event->attendee->get( get_current_user_id() ) : '',
+					'venue'                    => $venue_information
```
Comment (mauteri) [inline core/classes/class-rest-api.php:331]: `'venue'                    => $event->get_venue_information(),`
only using once, so no need to set variable; add trailing comma per standards.

File: assets/src/components/EventItem.js [hunk 3]
```diff
@@ -19,6 +19,8 @@ const EventItem = ( props ) => {
 
 	const eventClass = `gp-events-list`;
```
Comment (mauteri) [inline assets/src/components/EventItem.js:21]: remove linebreak

File: assets/src/components/EventItem.js [hunk 4]
```diff
@@ -19,6 +19,8 @@ const EventItem = ( props ) => {
...
 
+	const venue = event.venue;
```
Comment (mauteri) [inline assets/src/components/EventItem.js:22]: what do you think of something like...
``const venue = <a href="`${ venue.permalink }`">`${ venue.name }`</a>``

File: assets/src/components/EventItem.js [hunk 5]
```diff
@@ -35,6 +37,12 @@ const EventItem = ( props ) => {
 					>
 						<strong>{ event.datetime_start }</strong>
 					</div>
+					{ venue.name && (
+						<div>
+							Venue: { HtmlReactParser( '<a href=' + venue.permalink + '>' + venue.name + '</a>' ) }
```
Comment (mauteri) [inline assets/src/components/EventItem.js:42]: `Venue: { HtmlReactParser( venue ) }`

File: core/classes/class-rest-api.php (hunk 1 above)
Comment (mauteri) [reply core/classes/class-rest-api.php:319]: I would write a unit test for `get_venue_information` to ensure it doesn't fail in a way we don't expect.

**Review by mauteri — CHANGES_REQUESTED** [review]

**Review by mauteri — APPROVED** [review]

**Pull request merged**